src/doc/trpl/static.md

```diff
@@ -1,3 +1,47 @@
 % `static`
 
-Coming soon!
+Rust provides a ‘global variable’ sort of facility in static items. They’re
+similar to [constants][const], but static items aren’t inlined upon use.
```

Comment:
It may be worth mentioning high up here that the defining point of a `static` is that it has a fixed location in memory and there's only one of them.